Brocfile.js

```diff
@@ -86,7 +82,10 @@ const testBundle = watchify(merge([
   browserify: { debug: true, entries: ['./test/index.js'] }
 });
 
-const header = stew.map(find('config/versionTemplate.txt'), content => content.replace(/VERSION_PLACEHOLDER_STRING/, version()));
+const header = map(
+  find('config/versionTemplate.txt'),
```

Comment:
I'm working on deprecating `stew.find`, can you update this to use a `broccoli-funnel`?

Reply:
ok

Reply:
done